static/js/App.tsx

```diff
@@ -33,6 +34,10 @@ const App: FC = () => {
         path="/ui/:project/instances"
         element={<ProtectedRoute outlet={<InstanceList />} />}
       />
+      <Route
+        path="/ui/:project/instances/1/create"
```

Comment:
That `1` added on purpose, right?

Reply:
Yes, to reuse the /instances path prefix but ensure we don't block a detail page for an instance. Because instance names can't start with a number. It's a bit wired maybe, but I didn't find a better solution.

Reply:
What about `/instances?create` or `/instances?new`?

Reply:
With `?` the URL is also not so nice either.

I'd like to use `/instance-create`, but I don't know how to convince the `NavLink` from react-router for `/instances`, that it should also be active for a URL that does not match its own target.

Reply:
@edlerd , what if you make it `/instances/new` and put that route before this one https://github.com/canonical/lxd-ui/blob/main/static/js/App.tsx#L36 ? Will the router catch it up earlier?

Reply:
Yeah, that will work, but I'd like to avoid hogging the detail page for an instance with the name `new`, hence the workaround with the `/1`, because instances names can't start with a number.

Reply:
It's very hacky, and confusing for the user, to have that `/1/` in the path. I honestly find it more natural to have the `?` in the URL - what do you think @yurii-vasyliev ?

Reply:
Using the /1/ might mislead the user into thinking there is some sort of additional grouping. I'd try to avoid this approach. Maybe have something like "create-new" or "create-new-instance" can prevent us from having an issue?

Reply:
true, lets keep it simple. if someone has an instance with the name create-new it would be wired anyways.